docs/instrumentation-list.yaml

```diff
@@ -872,12 +1156,17 @@ methods:
   instrumentations:
   - name: methods
     srcPath: instrumentation/methods
+    scope:
+      name: io.opentelemetry.methods
     target_versions:
-      javaagent: []
+      javaagent:
+      - Java 8+
```

Comment:
this one isn't really targeting JDK classes, so maybe it should just be empty?

Reply:
do you think this one is worth including at all?

Reply:
I'd be fine either way, it sort of makes sense to me to keep it, and it sort of makes sense to me to remove it 😅